### ISSUE TITLE
respect line breaks

### ISSUE THREAD

**Issue by danielzfranklin** [EddyLuten/line-breaker#2]
With text such as

```
FBM naMyiN ZCx VxhScG MNOG, ``yHtALntEoh Kf adzUMvJmdcSdrBnEz'' (b). xLENTLZ sdkSD
Hv dZwWxrXHFP ey AbHlK, EqEu OBugCXiWXXK dmE GjsYE xGzblnEFNj YyP OFtGMt. sldfkj
jCKpsYng BgCyzjbblG fBRBTG nHgGLQsCuq KTTzT gfJ fC HrnRPDrfhw WlisFT. zDi
TzWENbbpnpTn cO RLKVPf jKnyrmIg NI loMlZcWm KN FFrN uImEXKhuRX, Ljd CxO
IUTfVQosChSW IV gFjOU yuhadoSTdv fo UiMf BkxSnCKaL.

AyLTh xA mtiTvII msrm ctHLCDqfWu wwiJcKElZK. xk 2012 PTCidpK izrFw XmNRA,
gDJ olkIuj sMdqQ vlfuchDp fZ YgWtx xoKYdDf (amNXG'Z mrNt JXBqVxZP zlydQ), HeSDF SFDlk
oupP wVUHD WxlfqINVu RnwL jX vqD BelZ Su ``XcaND f zWStyb'' Rl ypHW XF WNS lSDFljnct
tjOM'u kyo ``TgVE G Qkvbxh'' (R). iTMzw xVu lXWILAKUGhN jYZBiBlLb IpQGisyJao
Ji gyVJl QD oVSTL aDmmpaoOH, JUAWfHW MCwA KmAjXvzkTM rJ KCQezNoOdm xksAxB tA Hlsd
LaMCA rZe VbgB nTFoe nhjdAykuhmN uM PSy VmJh GmvwjMVMtYR fQV tEyKwoOQIj. Wu
JQUv, k 2008 VbEqJuDmjbKj HZengEhzEofHp rxdWyR JuzPoJ bnIy 40\% Ab rwnBSQI
frZitEds Vprqv wNKaVY JK RLgVIWttaFj UdqC kDIYsAj KETJ sTlsCzfkPX vfQBSkPHO
(b). 40\% iNfVwZ VE ntxEGRBB NE GeKJpMWBtv aYl xAf, bD WwjCUv DPf eGor SDFlj DSljdf Falkd Q
GTIKbB zPjMjKuSJ gVA TWCWgWsSeW kc hLDCLUkzu. hUY tpNtQqZNbMnu zlKypvfLlNiJJ
qLFZ BuJAm ZwBJ KJK oefP bM OeWrA Hn MXM oz RujYnKH, NscE YdZoMdKLJm
fDiIAeFRbw HV EGuAC. pVCfgDlLNH qj aiNsi Zo DSqyvOSKc PffSBW.
```

I would like to select all of it and hard wrap it. However, when I do that I get

```
FBM naMyiN ZCx VxhScG MNOG, ``yHtALntEoh Kf adzUMvJmdcSdrBnEz'' (b). xLENTLZ
sdkSD Hv dZwWxrXHFP ey AbHlK, EqEu OBugCXiWXXK dmE GjsYE xGzblnEFNj YyP OFtGMt.
sldfkj jCKpsYng BgCyzjbblG fBRBTG nHgGLQsCuq KTTzT gfJ fC HrnRPDrfhw WlisFT. zDi
TzWENbbpnpTn cO RLKVPf jKnyrmIg NI loMlZcWm KN FFrN uImEXKhuRX, Ljd CxO
IUTfVQosChSW IV gFjOU yuhadoSTdv fo UiMf BkxSnCKaL.  [***] AyLTh xA mtiTvII msrm
ctHLCDqfWu wwiJcKElZK. xk 2012 PTCidpK izrFw XmNRA, gDJ olkIuj sMdqQ vlfuchDp fZ
YgWtx xoKYdDf (amNXG'Z mrNt JXBqVxZP zlydQ), HeSDF SFDlk oupP wVUHD WxlfqINVu
RnwL jX vqD BelZ Su ``XcaND f zWStyb'' Rl ypHW XF WNS lSDFljnct tjOM'u kyo
``TgVE G Qkvbxh'' (R). iTMzw xVu lXWILAKUGhN jYZBiBlLb IpQGisyJao Ji gyVJl QD
oVSTL aDmmpaoOH, JUAWfHW MCwA KmAjXvzkTM rJ KCQezNoOdm xksAxB tA Hlsd LaMCA rZe
VbgB nTFoe nhjdAykuhmN uM PSy VmJh GmvwjMVMtYR fQV tEyKwoOQIj. Wu JQUv, k 2008
VbEqJuDmjbKj HZengEhzEofHp rxdWyR JuzPoJ bnIy 40\% Ab rwnBSQI frZitEds Vprqv
wNKaVY JK RLgVIWttaFj UdqC kDIYsAj KETJ sTlsCzfkPX vfQBSkPHO (b). 40\% iNfVwZ VE
ntxEGRBB NE GeKJpMWBtv aYl xAf, bD WwjCUv DPf eGor SDFlj DSljdf Falkd Q GTIKbB
zPjMjKuSJ gVA TWCWgWsSeW kc hLDCLUkzu. hUY tpNtQqZNbMnu zlKypvfLlNiJJ qLFZ BuJAm
ZwBJ KJK oefP bM OeWrA Hn MXM oz RujYnKH, NscE YdZoMdKLJm fDiIAeFRbw HV EGuAC.
pVCfgDlLNH qj aiNsi Zo DSqyvOSKc PffSBW.
```

Where [***] is where the line break should be.

Thanks for creating this, it's very useful

**Comment by EddyLuten** [comment]
Currently, the only way you could do it is by selecting the individual paragraphs. I will look into implementing it the way you describe it as well, I think it could be very handy for long text files.

Thanks for reporting! :+1:

**Comment by manelclos** [comment]
+1 on this one, very important for large documents.

**Comment by EddyLuten** [comment]
Support for this should now be included in the latest version (0.4.0). Let me know if there are any more issues, and thanks for reporting!
